integration-tests/testconfig/vrfv2plus/config.go

```diff
@@ -45,10 +45,14 @@ type General struct {
 	SubscriptionFundingAmountNative   *float64 `toml:"subscription_funding_amount_native"`     // Amount of LINK to fund the subscription with
 	SubscriptionRefundingAmountNative *float64 `toml:"subscription_refunding_amount_native"`   // Amount of LINK to fund the subscription with
 	FulfillmentFlatFeeNativePPM       *uint32  `toml:"fulfillment_flat_fee_native_ppm"`        // Flat fee in ppm for native currency for the VRF Coordinator config
-	FulfillmentFlatFeeLinkPPM         *uint32  `toml:"fulfillment_flat_fee_link_ppm"`          // Flat fee in ppm for LINK for the VRF Coordinator config
 	FulfillmentFlatFeeLinkDiscountPPM *uint32  `toml:"fulfillment_flat_fee_link_discount_ppm"` // Flat fee discount in ppm for LINK for the VRF Coordinator config
 	NativePremiumPercentage           *uint8   `toml:"native_premium_percentage"`              // Native Premium Percentage
 	LinkPremiumPercentage             *uint8   `toml:"link_premium_percentage"`                // LINK Premium Percentage
+
+	//Wrapper config
+	CoordinatorGasOverheadPerWord  *uint16 `toml:"coordinator_gas_overhead_per_word"`
+	WrapperNativePremiumPercentage *uint8  `toml:"wrapper_native_premium_percentage"`
```

Comment:
I believe these are called coordinatorNativePremiumPercentage and coordinatorLinkPremiumPercentage.
We got rid of WrapperPremium's in a recent change to VRFV2PlusWrapper contract

Reply:
We do have coordinatorNativePremiumPercentage and coordinatorLinkPremiumPercentage in wrapper config though for the wrapper to calculate gasCost of a request. I see that you removed these values.

Reply:
no, its still there 
* [here](https://github.com/smartcontractkit/chainlink/pull/12898/files#diff-462ae51e1ac043f7b3e100f53bb5948e3f6f9eab92991491d66c2a02f21f5d1eR299-R300)
* [here](https://github.com/smartcontractkit/chainlink/pull/12898/files#diff-6c84c5e0ce85553854ba9eb96d0bfa53738b872b3a31ff175a7a2cb3f413c18fR54-R55)

Reply:
Wrapper contract has separate coordinatorNativePremiumPercentage and coordinatorLinkPremiumPercentage variables